third_party/ijar/classfile.cc

```diff
@@ -1683,6 +1684,14 @@ static ClassFile *ReadClass(const void *classdata, size_t length) {
       // declaring compilation unit
       continue;
     }
+
+    // Mark non-abstract methods native since we drop the "Code attribute" and
+    // JVMS 4.7.3 says every method is either native or abstract if it doesn't have
+    // the "Code attribute"
+    if ((method->access_flags & ACC_ABSTRACT) != ACC_ABSTRACT) {
```

Comment:
We need to exclude interface methods, since they can't be marked as `ACC_NATIVE`, according to [JVMS 4.6](https://docs.oracle.com/javase/specs/jvms/se11/html/jvms-4.html#jvms-4.6):

>Methods of interfaces may have any of the flags in Table 4.6-A set except ACC_PROTECTED, ACC_FINAL, ACC_SYNCHRONIZED, and ACC_NATIVE (JLS §9.4). In a class file whose version number is less than 52.0, each method of an interface must have its ACC_PUBLIC and ACC_ABSTRACT flags set; in a class file whose version number is 52.0 or above, each method of an interface must have exactly one of its ACC_PUBLIC and ACC_PRIVATE flags set.